src/Serilog.Extensions.Logging/Extensions/Logging/SerilogLoggerProvider.cs

```diff
@@ -14,13 +15,19 @@ namespace Serilog.Extensions.Logging;
 /// </summary>
 [ProviderAlias("Serilog")]
 public class SerilogLoggerProvider : ILoggerProvider, ILogEventEnricher, ISupportExternalScope
+#if NET6_0_OR_GREATER
```

Comment:
The Serilog projects define constants for each conditional feature centrally in the CSPROJ, e.g.:

https://github.com/serilog/serilog/blob/dev/src/Serilog/Serilog.csproj

Could we define `FEATURE_ASYNCDISPOSABLE` and use it in place of the version constraints in the code?

Reply:
Uh, dooh. Sorry about that. I had noticed that the Serilog core code base used such flags, but forgot to check this code. I will address.

Reply:
I have defined FEATURE_ASYNCDISPOSABLE now